cluster_experiments/experiment_analysis.py

```diff
@@ -1,10 +1,10 @@
 from abc import ABC, abstractmethod
 from typing import List, Optional
-
+import warnings
```

Comment:
have you run pre-commit install? I believe this is not following the isort behaviour

Reply:
I don't see isort in pre commit 
![image](https://user-images.githubusercontent.com/33734080/216027611-3beac4cf-b5ec-43ad-bb0f-1f4b8c651047.png)

Reply:
ah damn, perhaps it is my linter, will investigate

Reply:
indeed I have it on my linter, could you add it in here?

Reply:
sure, done